Android/app/src/main/java/io/github/project_travel_mate/login/LoginPresenter.java

```diff
@@ -88,6 +88,7 @@ public void onFailure(Call call, IOException e) {
             public void onResponse(Call call, final Response response) throws IOException {
 
                 final String res = Objects.requireNonNull(response.body()).string();
+                final String msg = "signup succeeded! please login";
```

Comment:
why? This will mislead the users into thinking that sibnup has succedded, event when it isnt.